### PR TITLE
[PropertyAccess] Adds entries to CHANGELOG and UPGRADE

### PR DESCRIPTION
**Pull request by jschaedl**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | no     
| Deprecations? | yes 
| Tests pass?   | yes    
| Fixed tickets | #32179   
| License       | MIT
| Doc PR        | N/A

In PR #32241 I missed to add entries to the CHANGELOG and UPGRADE file.

### CODE REVIEW

File: UPGRADE-4.4.md [hunk 1]
```diff
@@ -76,6 +76,11 @@ MonologBridge
 
  * The `RouteProcessor` has been marked final.
 
+PropertyAccess
+--------------
+
+ * Deprecated to pass `null` as `$defaultLifetime` 2nd argument of `PropertyAccessor::createCache()` method, pass 0 instead.
```
Comment (GuilhemN) [inline UPGRADE-4.4.md]: `passing`?

Comment (jschaedl) [reply UPGRADE-4.4.md]: Hmm, maybe... @ocrampete16 Can you help out?

Comment (ocrampete16) [reply UPGRADE-4.4.md]: "Deprecated passing `null`" is correct. 👍

```suggestion
 * Deprecated passing `null` as 2nd argument of `PropertyAccessor::createCache()` method (`$defaultLifetime`), pass 0 instead.
```

Comment (jschaedl) [reply UPGRADE-4.4.md]: Thanks Marco :-)

File: UPGRADE-4.4.md [hunk 2]
```diff
@@ -76,6 +76,11 @@ MonologBridge
 
  * The `RouteProcessor` has been marked final.
 
+PropertyAccess
+--------------
+
+ * Deprecated passing `null` as 2nd argument of `PropertyAccessor::createCache()` method (`$defaultLifetime`), pass 0 instead.
```
Comment (OskarStark) [inline UPGRADE-4.4.md]: 0 in ticks?

Comment (jschaedl) [inline UPGRADE-4.4.md]:
```suggestion
 * Deprecated passing `null` as 2nd argument of `PropertyAccessor::createCache()` method (`$defaultLifetime`), pass `0` instead.
```

File: src/Symfony/Component/PropertyAccess/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+4.4.0
+-----
+
+ * deprecated passing `null` as `$defaultLifetime` 2nd argument of `PropertyAccessor::createCache()` method,
+   pass 0 instead
```
Comment (jschaedl) [inline src/Symfony/Component/PropertyAccess/CHANGELOG.md]:
```suggestion
   pass `0` instead
```

File: UPGRADE-4.4.md (hunk 2 above)
Comment (jschaedl) [reply UPGRADE-4.4.md]: Fixed

**Comment by ogizanagi** [comment]
You could add an entry in UPGRADE-5.0.md as well mentioning the support for `null` was dropped.

**Comment by jschaedl** [comment]
@ogizanagi here you go: https://github.com/symfony/symfony/pull/32234/files#diff-6a06298e7799bf3b02d99b4659f7bff6R341

**Review by OskarStark — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @jschaedl.

**Pull request merged**